coalib/settings/ConfigurationGathering.py

```diff
@@ -1,7 +1,9 @@
 import os
+import shutil
```

Comment:
`ConfigurationGathering.py` seems to be a weird place for this functionality to be implemented in. This is an action and should probably be handled at an higher level.